.golangci.yml

```diff
@@ -8,6 +8,7 @@ linters:
   enable:
     - godot
     - gofmt
+    - goimports
```

Comment:
Having `goimports` and `gci` seems redundant to me, maybe we should settle just on one.